docs/integrations-guides/v1/dispersal/clients/eigenda-proxy-v1.md

```diff
@@ -0,0 +1,27 @@
+# EigenDA Proxy (V1)
+
+## About
+See [this page](../../../eigenda-proxy.md) for a high level description of the proxy.
+
+### Key Releases
+| version | (mainnet) max blob size | (holesky) max blob size | supported stacks   | Cert version(s) |
+|---------|-------------------------|-------------------------|--------------------|-----------------|
+|  v1.3.1 |        `2mib`           |          `2mib`         |      OP Stack      |      v0         |
+|  v1.4   |        `4mib`           |          `16mib`        |      OP Stack      |      v0         |
+|  v1.4.1 |        `16mib`          |          `16mib`        |  OP Stack, Orbit   |      v0         |
```

Comment:
curious if this table is something we still wanna maintain
cc @samlaf, @mmurrs

Reply:
I don't think so personally. It's already super outdated. Prefer we keep thsi kind of info as part of the release notes, and point people to those